lib/rucio/core/did_meta_plugins/filter_engine.py

```diff
@@ -271,13 +272,17 @@ def _try_typecast_string(self, value):
             pass
         return value
 
-    def create_mongo_query(self, additional_filters={}):
+    def create_mongo_query(
+        self,
+        additional_filters: Optional[list[tuple[str, Callable, str]]] = None
```

Comment:
`Iterable` instead of `list` (`additional_filters` is only used in a for loop)

Reply:
Changed, now

```
additional_filters: Optional[Iterable[tuple[str, Callable, str]]] = None
```